{"code": "warn(\"The data contains more than 10,000 rows.\")", "compilation_passed": true, "time": 0.0035631656646728516, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "warn('The data contains more than 10,000 rows.')", "compilation_passed": true, "time": 0.0021920204162597656, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "warn(\"The data contains more than 10000 rows.\")", "compilation_passed": true, "time": 0.0020029544830322266, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "filterwarnings(\"always\", message=\"The data contains more than 10000 rows.\",", "compilation_passed": false, "time": 0.00011682510375976562, "tests_passed": false, "error": "SyntaxError"}
{"code": "simplefilter(\"always\")", "compilation_passed": true, "time": 0.0016961097717285156, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "warn(\"The data contains more than 10000 rows.\", UserWarning)", "compilation_passed": true, "time": 0.0014929771423339844, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "warn('The data contains more than 10000 rows.')", "compilation_passed": true, "time": 0.0013849735260009766, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "filterwarnings(\"always\")", "compilation_passed": true, "time": 0.0013620853424072266, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "simplefilter('always')", "compilation_passed": true, "time": 0.0013680458068847656, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": "warn(", "compilation_passed": false, "time": 8.296966552734375e-05, "tests_passed": false, "error": "SyntaxError"}
